### ISSUE TITLE
'host' parameter in BaseStorage's default_settings __init__() is set to None

### ISSUE THREAD

**Issue by Kahdeg-15520487** [jschneier/django-storages#932]
requirements.txt
```django-storages==1.10```

settings.py:
```
SFTP_STORAGE_HOST = 'IP'
SFTP_STORAGE_ROOT = '~/'

DEFAULT_FILE_STORAGE = 'storages.backends.sftpstorage.SFTPStorage'
STATICFILES_STORAGE = 'storages.backends.sftpstorage.SFTPStorage'
SFTP_STORAGE_PARAMS = {
    "username" : "sftpuser1"
}
```
I added print statement to storages/base.py:
```
class BaseStorage(Storage):
    def __init__(self, **settings):
        default_settings = self.get_default_settings()

        print("=====")
        for name, value in default_settings.items():
            if not hasattr(self, name):
                print("set "+name+" to "+str(value))
                setattr(self, name, value)
        print("=====")

        print("=====")
        for name, value in settings.items():
            if name not in default_settings:
                raise ImproperlyConfigured(
                    "Invalid setting '{}' for {}".format(
                        name,
                        self.__class__.__name__,
                    )
                )
            setattr(self, name, value)
            print("set "+name+" to "+str(value))
        print("=====")
```

Resulting logs:
```
=====
set host to IP
set params to {'username': 'sftpuser1'}
set interactive to False
set file_mode to None
set dir_mode to None
set uid to None
set gid to None
set known_host_file to None
set root_path to ~/
set base_url to /
=====
=====
set host to None
=====
```

**Comment by jschneier** [comment]
Can you verify this is fixed on master? Sorry, I should have released it already.

**Comment by jschneier** [comment]
This was fixed in #926. As soon as my PR for the S3 issue landed I will release a bugfix version. Sorry about that.
